src/server/system_services/bucket_server.js

```diff
@@ -73,6 +73,9 @@ function new_bucket_defaults(name, system_id, tiering_policy_id, owner_account_i
         object_lock_configuration: config.WORM_ENABLED ? {
             object_lock_enabled: lock_enabled ? 'Enabled' : 'Disabled',
         } : undefined,
+        encryption: {
```

Comment:
The problem with setting it like this here is that it also applies for namespace buckets - which we don't know for sure if they have encryption set or not. So I think you need to check here if it's data buckets only (no namespace). For namespace that can be a bigger discussion on what we should return, as it can be a mix of resources of different kinds. Would handling only data buckets be enough for now?